src/shared/util.js

```diff
@@ -1130,6 +1130,19 @@ if (
   };
 }
 
+// TODO: Remove this once the `javascript.options.experimental.math_sumprecise`
+//       preference is removed from Firefox.
+if (
+  (typeof PDFJSDev === "undefined" || PDFJSDev.test("SKIP_BABEL || TESTING")) &&
```

Comment:
Would it perhaps be safer to unconditionally do the `typeof` check here, given the quite limited browser support? It's for example not entirely clear to me at least if this condition will also work for Chrome outside of testing environments, for example in the addon, because `PDFJSDev` is defined then but not `TESTING` nor `SKIP_BABEL` for modern builds, and if I read this code correctly we won't polyfill it then while the compatibility table lists the feature as unsupported in Chrome.

Reply:
The idea was that we'd only need this crude polyfill in "modern" build, and that we'd be able to rely on core-js in the `legacy` builds.
While a core-js polyfill do exist, it turns out that the feature is new enough that Babel won't actually include it :-(

For now we can use our polyfill, and use the core-js polyfill later on; thanks for spotting this!